mllib/src/main/scala/org/apache/spark/ml/evaluation/MulticlassClassificationEvaluator.scala

```diff
@@ -97,6 +98,7 @@ class MulticlassClassificationEvaluator @Since("1.5.0") (@Since("1.5.0") overrid
     case "recall" => true
     case "weightedPrecision" => true
     case "weightedRecall" => true
+    case "accuracy" => true
```

Comment:
I am fine with either way. I think your suggestion is good. I will make change to simply return true now. Thanks!